.github/workflows/pr-close-delete-env.yml

```
@@ -3,14 +3,19 @@ name: PR - Delete Environment when PR is closed
 on:
   pull_request:
     types: [closed]
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Pull request number"
+        required: true
+        default: ""
 
 defaults:
   run:
     shell: pwsh
 
 env:
-  ENVIRONMENT_NAME: "PR #${{ github.event.number }}"
-  SLOT_NAME: pr-${{ github.event.number }}
+  SLOT_NAME: pr-${{  github.event.inputs.environment == '' && github.event.number || github.event.inputs.environment }}
```

```suggestion
  SLOT_NAME: pr-${{  github.event.inputs.environment || github.event.number }}
```

if nullish coalescing is working can we do this instead?